cmgd/cmgd_frntnd_adapter.c

```diff
@@ -413,7 +429,9 @@ static int cmgd_frntnd_send_setcfg_reply(cmgd_frntnd_sessn_ctxt_t *sessn,
 
 	CMGD_FRNTND_ADPTR_DBG("Sending SET_CONFIG_REPLY message to CMGD Frontend client '%s'",
 		sessn->adptr->name);
-
+	cmgd_get_realtime(&sessn->adptr->set_cfg_stats.end,
```

Comment:
Rewrite this as... 
if (set_cfg_stats.enable)
    cmd_ger_realtime(&sessn->adptr->set_cfg_stats.end);

Reply:
Also why is 'set_cfg_stats.enable' recorded per session? This should be stored under cmd_master{}.